docs/content/interop.md

```diff
@@ -49,8 +49,8 @@ OutputPath=ExampleGame/Generated/
 ;Leave empty if no includes required.
 
 ;MyClass
-ExampleGameMyClass=ExampleGameMyClass.h
-ExampleGameMyClass=IncludePath/MyClassDependency.h
+/Script/ExampleGame.ExampleGameMyClass=ExampleGameMyClass.h
```

Comment:
Should we add a Jira ticket to look at this usability?

Reply:
I already have one, https://improbableio.atlassian.net/browse/UNR-466, as it will be done within the editor I intend to make the menus select from assets if possible. if not then I'll create a new ticket